### ISSUE TITLE
Test for WCAG success criterion 1.2.5: Audio Description (Prerecorded)

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#174]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-audio-desc-only.html
 Audio description is provided for all prerecorded video content in synchronized media.
## Scope selector

Video content
## Applicable

synchronized media
## Not applicable
- if all of the information in the video track is already provided in the audio track
## Relationship with other success criteria
- At Level A in Success Criterion 1.2.3, authors do have the choice of providing either an audio description or a full text alternative. 
- At level AA Under Success Criterion 1.2.5 authors must provide an audio description - a requirement already met if they chose that alternative for 1.2.3, otherwise an additional requirement. 
- At Level AAA under Success Criterion 1.2.8 they must provide an extended text description. This is an additional requirement if both 1.2.3 and 1.2.5 were met by providing an audio description only. If 1.2.3 was met, however, by providing a text description, and the 1.2.5 requirement for an audio description was met, then 1.2.8 does not add new requirements.
## Failures
- no failures defined
## Techniques
- G78: Providing a second, user-selectable, audio track that includes audio descriptions

OR
- G173: Providing a version of a movie with audio descriptions

OR
SC1.2.8 (G8: Providing a movie with extended audio descriptions)

OR if a talking head video:
- G203: Using a static text alternative to describe a talking head video
## Relationship between techniques and outcomes
- There are no failures defined, it's hard to detect an automatic failure
- n/a is hard to detect as 'if all of the information in the video track is already provided in the audio track' is difficult to analyse
- We can provide a pass if we can detect a audiotrack (.mp3) and a 'cantTell' if not?
